cmd/complete.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/vmware/service-level-indicator-exporter-for-kafka/config"
+	"github.com/vmware/service-level-indicator-exporter-for-kafka/pkg/kafka"
+)
+
+var completeCmd = &cobra.Command{
+	Use:     "complete",
+	Aliases: []string{"complete"},
```

Comment:
I think we have to add a note saying that running producer and consumer in the same pod is not the preferred approach for many requests. Use it at your own risk.